src/helloworld/README.md

```diff
@@ -99,6 +99,9 @@ root@cheribsd:~ # file helloworld
 helloworld: ELF 64-bit LSB pie executable, ARM aarch64, C64, CheriABI, version 1 (SYSV), dynamically linked, interpreter /libexec/ld-elf.so.1, for FreeBSD 14.0 (1400064), FreeBSD-style, with debug_info, not stripped
 ```
 
+To target the [Benchmark ABI](../benchmarking/), add the argument
+`-mabi=purecap-benchmark` to the `cc` command line.
```

Comment:
Equivalent file(1) output?

Reply:
The version of file(1) I have right now doesn’t show the difference. I need to reinstall it to a new snapshot tonight!